### DISCUSSION TITLE
Error:  Invalid `prisma.post.create()` invocation:   Unable to require(`/home/joseph/Documents/life/firebase/node_modules/.pnpm/@prisma+client@4.14.1_prisma@4.14.1/node_modules/.prisma/client/libquery_engine-debian-openssl-3.0.x.so.node`).

### DISCUSSION THREAD

**Discussion by fhub-1** [prisma/prisma#19509] [Q&A] [answered]
![Screenshot from 2023-05-29 16-38-58](https://github.com/prisma/prisma/assets/71527527/6cd86a92-4fe6-4859-8bd3-e0ac4242c40c)

**Comment by fhub-1** [comment, marked as answer]
i manged to fix the error kindly check out here for the solution too solve the glibc thing https://github.com/prisma/prisma/issues/9174

**Reply by nurul3101** [reply]
Thanks for letting us know and sharing the link. 🙂
